kpi-tracker/src/main.rs

```diff
@@ -50,8 +50,47 @@ struct BlockDetails {
     height: AbsoluteBlockHeight,
 }
 
-type AccountsTable = HashMap<AccountAddress, AccountDetails>;
+/// Holds selected attributes about accounts created on chain.
+#[derive(Debug)]
+struct AccountDetails {
+    /// Whether an account was created as an initial account or not.
+    is_initial: bool,
+    /// FK to the block in which the account was created.
+    block_hash: BlockHash,
+}
+
+/// Holds selected attributes of an account transaction.
+#[derive(Debug)]
+struct TransactionDetails {
+    /// The transaction type of the account transaction
+    transaction_type: TransactionType,
```

Comment:
What do you do with the "unknown" one?

Reply:
Stored this as an option, which I'm thinking will translate to `null` in the database.

Reply:
Possibly :) You'll have to define the storage type for TransactionType though.